lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -38,6 +47,25 @@ spec = do
         eventually $ do
             r <- request @[ApiStakePool] ctx listStakePoolsEp Default Empty
             expectResponseCode HTTP.status200 r
+            -- With the current genesis.yaml we have 1 pool with 1 lovelace, and
+            -- an epoch length of 3, meaning it will have produced either 1 or
+            -- 2 blocks in the current epoch.
+            verify r
+                [ expectListSizeEqual 1
+                , expectListItemFieldEqual 0
+                    (metrics . stake) (Quantity 1)
+                , expectListItemFieldBetween 0
+                    (metrics . blocks) (Quantity 1, Quantity 2)
+                ]
+        -- We should be able to catch the stake-pool data in an un-synced state
+        -- when we enter into a new epoch. The results should then be
+        -- unavailible.
+        --
+        -- This might take a few tries (epoch changes), so it is only feasible
+        -- to test with very short epochs.
+        eventually $ do
+            r <- request @[ApiStakePool] ctx listStakePoolsEp Default Empty
+            expectResponseCode HTTP.status503 r
```

Comment:
Would be nice to have a nice message returned here explaining why it is `503`...

Reply:
We need to make sure that the polling delay in eventually is short in front of the stepping delay of the networking layer, otherwise, it might take a lot of tries :/ 

Also, on a different note, this should really be another test scenario.

Reply:
There is a message, if that's what you mean! (see gifs in description of https://github.com/input-output-hk/cardano-wallet/pull/881)

```
I can't list stake pools yet because I need to scan the blockchain for metrics first. I'm at 0%
```

_Could maybe check that the message ends with a "%", to keep it simple 🤔 _

Reply:
Good! Then you can use:
```
expectErrorMessage "I can't list stake pools yet because I need to scan the blockchain for metrics first. I'm at"
```
(it actually checks if message contains text, not if the message is exactly like ...)

Reply:
> Also, on a different note, this should really be another test scenario.

I wasn't sure the same jormungandr instance was used for all the scenarios, but I see it is. Then it makes sense 👍 (I want to make sure it returns 200 _before_ it returns 503 again)